Source/UI/Hashtag/HashtagListView.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  HashtagListView.swift
+//  Planetary
+//
+//  Created by Martin Dutra on 30/11/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import Analytics
+import CrashReporting
+import Logger
+import SwiftUI
+
+struct HashtagListView: View {
+
+    @EnvironmentObject
+    private var botRepository: BotRepository
+
+    @EnvironmentObject
+    private var helpDrawerState: HelpDrawerState
+
+    private let helpDrawer = HelpDrawer.hashtags
+
+    @State
+    private var allHashtags: [Hashtag]?
+
+    @State
+    private var filteredHashtags: [Hashtag]?
+
+    @State
+    private var searchText = ""
+
+    @SwiftUI.Environment(\.isSearching)
+    private var isSearching
```

Comment:
I think this could be a @State instead of @Environment? Then it won't be passed down to subviews.

Reply:
👌

Reply:
It looks like this didn't get changed?

Reply:
I thought you were talking about searchText. I don't really understand what you mean by that because
```
@SwiftUI.Environment(\.isSearching)
private var isSearching
```
is a variable set by the environment automatically (https://developer.apple.com/documentation/swiftui/environmentvalues/issearching), can I change that to @State ?

Reply:
Oh! I didn't realize it was something defined by the system. I thought you were adding that value to the environment yourself.